microsoft-edge/webview2/reference/javascript/chrome.yml

```diff
@@ -0,0 +1,23 @@
+### YamlMime:TSType

```

Comment:
Global fix : The word "Reference" appears under the H1 on every page.

Reply:
I think it just means the page is showing Reference content? I see this in our .NET docs as well.

https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core?view=webview2-dotnet-1.0.1462.37
![image](https://user-images.githubusercontent.com/55165503/211422748-850ad3b5-0c97-4cf1-82c0-75a8a9880f62.png)